src/eth/abis/lockProxy.json

```diff
@@ -1,588 +1,109 @@
 [
   {
```

Comment:
please do not replace the lockproxy contract abi with your bridge entrance contract abi, they are different things and should use different json files.